app/config/messagequeue.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+return [
+  'enabled' => false,
+  'type' => 'none', // none, rabbitmq
+  'rabbitmq' => [
+    'host'=> 'localhost',
+    'port'=> 5672,
+    'username'=> 'guest',
+    'password'=> 'guest',
+    'vhost'=> '/',
+    'exchange'=> 'learninglocker',
+    'ssl_enabled'=> true,
+    'ssl'=> [
+      'cafile'=> '/etc/ssl/cert/somecert.pem',
+      'verify_peer'=> true
+    ]
+  ],
+  'model_keys' => ['id', 'lrs_id', 'client_id', 'statement', 'stored']
```

Comment:
Add a second array that tracks what's included from the statement? My only concern is there should be a way to ask for all statement first order keys, otherwise we run in to future proofing issues.